planet4-gutenberg-blocks.php

```diff
@@ -134,7 +134,6 @@
 	'planet4-blocks/media-video',
 	'planet4-blocks/social-media',
 	'planet4-blocks/spreadsheet',
-	'planet4-blocks/submenu',
```

Comment:
We do need to keep these here for now, to make sure we select which blocks are allowed per post type. There is a ticket to move this filter to the theme.

Reply:
Ah it's actually your [ticket](https://jira.greenpeace.org/browse/PLANET-7615) haha

Reply:
Let's start by working-reviewing-merging [that ticket](https://jira.greenpeace.org/browse/PLANET-7615), then.

Reply:
Why? When moving this filter to the theme it will also need to keep the Submenu block in it

Reply:
So if you just keep the submenu block in this filter, we can merge the PR already, the rest looks good to me 👍